scripts/secure-boot/generate-images/generate-image.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+SCRIPTDIR=$(readlink -f $(dirname "$0"))
```

Comment:
```
In scripts/secure-boot/generate-images/generate-image.sh line 3:
SCRIPTDIR=$(readlink -f $(dirname "$0"))
                        ^------------^ SC2046 (warning): Quote this to prevent word splitting.
```

Reply:
fixed [here](https://github.com/Dasharo/open-source-firmware-validation/pull/192/commits/bce10d72eb89079d32fb6db71846782fa2f3e593#diff-00f53487a7d91b72da783fcf7286011bd2c964cb60496c9c6e607a55e9ae748dR5), pre-commit passes